src/clj_jwt/base64.clj

```diff
@@ -46,8 +50,9 @@
 
 ;; URL-Safe Decoder
 (defn url-safe-decode
+  ^bytes
   [^String s]
-  (-> (case (mod (count s) 4)
+  (-> (condp = (mod (.length s) 4)
```

Comment:
Why do you prefer `condp =` here I think `case` is faster and preferable to use when possible.